content/en/service_management/app_builder/components/custom_charts.md

```diff
@@ -42,7 +42,7 @@ The following example shows how to create a histogram chart illustrating Datadog
 
 1. Click **+ All Components** and select **Custom Chart** to add a component called `customChart0`.
 1. Click **Show Chart Examples**.
-1. Select **Simple Histogram** and click **Confirm**.<br>The following value populates in the Vega Specification:
+1. Select **Simple Histogram** and click **Confirm**.<br>The following value populates in the Vega-Lite Specification:
```

Comment:
Possible follow-up: `specification` can be lowercased, but I saw it was done twice in a row so I figured it might have been an intentional choice from the styleguide I was not familiar with.

Reply:
Ah, thanks for explaining the rationale for the public docs. I can see how the screenshots makes this confusing. It is good to discuss this before we GA the feature.

The relationship between the two languages is that Vega-Lite is converted to Vega underneath.

We are trying to encourage people to use Vega-Lite rather than Vega because it is significantly more concise while granting similar expressivity for most cases, and will yield better results when searching (since Vega-Lite is fairly unique to the graphing technology, whereas Vega is also a star, a last name, etc). 

Even if we support Vega (the more verbose notation), I would still suggest our docs towards the "Vega-Lite" documentation rather than "Vega". (We are doing the same approach of preferring the Vega-Lite terminology for the [Wildcard Widget](https://docs.datadoghq.com/dashboards/widgets/wildcard/) ).

I would recommend that we update the UI text too. Let me know if we need to get the UI text changed before we update the public docs.

Reply:
I strongly prefer that we change the UI text before we change the docs. Otherwise it might confuse users.

Reply:
Thanks for the feedback, understood. I will ask about this in the team slack channel and get back to you once that has been updated.

Reply:
The internal copy has been updated to replace "Vega" with "Vega-Lite" 👍  

https://github.com/DataDog/web-ui/pull/173378